submit/samples/sample_checklist/sample_checklist_errors.md

```diff
@@ -0,0 +1,24 @@
+# Sample validation errors
+
+When you submit your samples and select to validate them against a checklist, you may get errors in your receipt based
+on the validation. As per January 2025, these errors are going to be deprecated, in favour of sample validation via the
+[BioSamples Database (BSD)](https://www.ebi.ac.uk/biosamples/). While the validation will be the same, error messages 
+will differ between the previous XML schema validation and the new JSON schema validation.
+
+Below is a table, compiling the comparison of all the possible errors related to checklist validation:
+
+| **XML validation (old)**                                                                                                                                                         | **BioSamples validation (New)**                                                                                                                                                            |
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Error at {<sample-name>}. Invalid units "{value-of-units}" for field "{<attribute-name>}". Allowed units are "{<list-of-allowed-values>}"                                        | sample:{<sample-name>}; attribute:{<path-to-attribute>}; reason:must be equal to one of the allowed values:{list-of-allowed-values}                                                        |
```

Comment:
I would add a space after all the colons, in all other error messages and in the app itself. Is there a reason for not using spaces after the colons?
```suggestion
| Error at {<sample-name>}. Invalid units "{value-of-units}" for field "{<attribute-name>}". Allowed units are "{<list-of-allowed-values>}"                                        | sample:{<sample-name>}; attribute: {<path-to-attribute>}; reason: must be equal to one of the allowed values: {list-of-allowed-values}                                                        |
```